source/vistas/ui/controllers/export.py

```diff
@@ -16,18 +16,26 @@
 
 
 class ExportTextCtrl(wx.TextCtrl):
+    """ Subclass of wx.TextCtrl with a reference to a text ExportItem. """
```

Comment:
"Subclass of wx.TextCtrl" is immediately apparent from the class definition and should not be duplicated in the docstring.